osc-server/src/main/java/org/osc/core/broker/service/persistence/VirtualSystemEntityMgr.java

```diff
@@ -185,7 +185,7 @@ public static synchronized Long generateUniqueTag(EntityManager em, VirtualSyste
     }
 
     public static boolean isProtectingWorkload(VirtualSystem vs) {
```

Comment:
can you add javadoc to these methods to make it clear what they consider protected workload?

org.osc.core.broker.service.persistence.DistributedApplianceEntityMgr.isProtectingWorkload(DistributedAppliance)
org.osc.core.broker.service.persistence.VirtualSystemEntityMgr.isProtectingWorkload(VirtualSystem)
org.osc.core.broker.service.persistence.DeploymentSpecEntityMgr.isProtectingWorkload(DeploymentSpec)

Reply:
please address this comment and add javadco.